### DISCUSSION TITLE
Area Card: Add Media-Player state (playing) to alarm class or as control

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#20987] [Other feature requests] [answered]
Hi, I would like to discuss the option to add a media player information / control to the Areal-Card...

Option 1:
Show an alarm icon if a media player in the area is playing

Option 2:
Add a controll to pause/stop a playing mediaplayer (similar to switch / light)
<img width="744" alt="image" src="https://github.com/home-assistant/frontend/assets/26391061/e5486514-1b89-42e3-8314-6d67ec3dd830">

**Comment by karwosts** [comment, marked as answer]
For #&zwnj;1 you can achieve it if you make a template binary sensor, maybe with `device_class: sound`, or `device_class: running`

#&zwnj;2 has had some discussion in the past: https://github.com/home-assistant/frontend/pull/13948
